README

```diff
@@ -1,9 +1,12 @@
 
- OpenSSL 1.1.1-pre5-dev
+ OpenSSL 1.1.1
 
- Copyright (c) 1998-2018 The OpenSSL Project
- Copyright (c) 1995-1998 Eric A. Young, Tim J. Hudson
- All rights reserved.
+ Copyright 2016-2018 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
That year is wrong - it should be 1998- ... to match usage elsewhere ...

Reply:
fixed, additional commit pushed.